.github/workflows/yocto-build-deploy.yml

```diff
@@ -501,7 +501,7 @@ jobs:
           compression-level: 7
           path: |
             ${{ env.DEPLOY_PATH }}/image/balena.img
-            ${{ env.DEPLOY_PATH }}/image/balena-image.docker
+            ${{ env.DEPLOY_PATH }}/balena-image.docker
```

Comment:
The hardcoded paths in config.js are very annoying

Reply:
@vipulgupta2048 we might need to make that configurable for another reason